providers/keycloak.go

```diff
@@ -61,6 +61,7 @@ func NewKeycloakProvider(p *ProviderData, opts options.KeycloakOptions) *Keycloa
 
 	provider := &KeycloakProvider{ProviderData: p}
 	provider.setAllowedGroups(opts.Groups)
+	provider.setAllowedACR(opts.ACRs)
```

Comment:
we don't support the legacy keycloak provider anymore and don't want to introduce new features to it. therefore please remove this line

```suggestion
```